strawberry/schema/schema.py

```diff
@@ -78,8 +79,9 @@ def __init__(
 
         graphql_types = []
         for type_ in types:
-            graphql_type = self.schema_converter.from_object(type_._type_definition)
-            graphql_types.append(graphql_type)
+            graphql_type = self.schema_converter.from_type(type_)
+            if isinstance(graphql_type, GraphQLNamedType):
+                graphql_types.append(graphql_type)
```

Comment:
why do we need this check?

Reply:
Good question!

I initially added it just to keep mypy happy :grimacing: 

but now that you brought it up, I've looked a bit more into it:

As far as I can tell, the only non-named type that could be returned here is a GraphQLList.

Adding `types=[List[MyType]]` would definitely be weird, but shouldn't be ignored. I'll replace it with an assertion instead

I'll replace this with an assertion instead

Reply:
yes, assertion would be better! thanks 😊